### DISCUSSION TITLE
Session start failed: clock not synchronized or other error

### DISCUSSION THREAD

**Discussion by hgreyh** [NightscoutFoundation/xDrip#1907] [Q&A] [answered]
Hello, I use dexcom g6 with replaceable battery, xdrip v.49817a5-2021.10.04, smartwatch android7.1.1. When i try to start sensor an error occurs:"Session start failed: clock not synchronized or other error". If I turn off the OB1 data collection service, I can start sensor and get my BG data. Where can I find information about OB1 work and this problem?

**Comment by Navid200** [comment, marked as answer]
Would you please take a snapshot of the system status page?  
Then, blackout the serial number except the first two characters.
Then, please post it here.  To do that, please start a post.  Then, tap on the bottom border and select the file to post.

**Reply by hgreyh** [reply]
![image1](https://user-images.githubusercontent.com/74299959/148303644-b3a9584f-d4fa-4cde-88f3-fd71a30cedfa.jpg)
Serial number 81....

**Comment by Navid200** [comment]
When the time comes to switch to a new sensor, some people place the new sensor on while the old one is still active.
Then, they stop sensor, remove and snap the transmitter into the new sensor and start.
This is not a good idea.

I always wait at least 10 minutes after stopping the sensor and removing the transmitter before snapping it into the new sensor.
Most of the start errors are caused by the transmitter and can be avoided by doing what I suggested.

To resolve the issue after it has happened, you can perform a  restart:
https://github.com/Navid200/xDrip/wiki/Restart-G6-Sensor

**Reply by hgreyh** [reply]
this is what i did: 0.enable engineering mode;  1.stop sensor;  2. wait 10 minutes;  3. remove the transmitter;  4. wait 15 minutes;  5.OB1 enable;  6.insert the sensor; 7.  wait 10 minutes until the stop sensor in system ststus;  8.remove transmitter;  9.wait 15 minutes;  10. insert the transmitter;  11. wait for connection;  12.start sensor.  i got the same errors: unusual session start failure and clock not sinchronized.
![20220110_104511](https://user-images.githubusercontent.com/74299959/148734838-60c7b8b0-1e72-4ea0-9c08-21237358338f.jpg)
![20220110_104449](https://user-images.githubusercontent.com/74299959/148734843-556110ed-61c9-4fd1-b841-a01cc7b0f9e9.jpg)

**Reply by Navid200** [reply]
You have replied to an older post of mine in this thread.

Is this still a problem?

**Reply by hgreyh** [reply]
I used all your recommendations but i still have the same problem

**Reply by Navid200** [reply]
What triggered this?

Did it happen when you switched to a new phone?
Or, did you bump the transmitter?

I mean this cannot be the first time you are using G6, is it?

**Reply by hgreyh** [reply]
I have encountered this error once before.  I changed the transmitter and the error was gone.  now i cant use OB1.  this happens both when starting a new sensor and when restarting an old one.  this happens with two transmitters after replacing the battery.  possibly the trigger is the replacement of the battery.  perhaps the trigger is the receiver watch

**Reply by hgreyh** [reply]
After hard reset transmitter i had no problem with start new sensor. Why did this happen?

**Reply by Navid200** [reply]
There are so many factors involved.
xDrip, the phone, the transmitter, and the sensor.
A lot relies on the experiences of users with different phones.
Most of the users use new G6 transmitters.

But, what makes this even more complicated is that the transmitter is not a standard transmitter.  I mean the fact that the battery has been replaced.  So, what the voltage of the battery is and how well it is making contact with the transmitter are additional factors that make this a puzzle.
It's very hard for anyone tp diagnose this and state for sure what could have caused the problem.

If you can show that an older version of xDrip works fine and a newer version of xDrip fails, that would be evidence that it's xDrip that is the cause.
But, without that, we can only speculate what the problem may be.

**Reply by hgreyh** [reply]
is this not an OB1 bug?  if OB1 is disabled the sensor starts successfully.  what does the message mean: session start failed info:1970-01-01... and clearing stored timing sync information for 81...(sn transmitter)
![clear](https://user-images.githubusercontent.com/74299959/149639856-ce71fc06-c275-444b-bd2b-9afc515fd3f9.jpg)
![stop](https://user-images.githubusercontent.com/74299959/149639860-60045ade-ec1f-4e9c-8ba0-2ed606942d56.png)

**Reply by Navid200** [reply]
No, it is not an OB1 bug.

You should decide if you want to benefit from G6 factory calibration or not.
If yes, you need to enable OB1 and the error is due  to the transmitter, not xDrip.

If you don't care for factory calibration, you can disable Native Algorithm.  You should still enable OB1.  Then, there will be no errors and the rules will be relaxed.  

However, factory calibration is the main advantage of G6 over G5.  So, I strongly advise that you use it.

**Comment by Navid200** [comment]
Since you have an older transmitter, it transmits raw values.
That's why you can use it in non-native mode.  
This provides some pros and cons of the two modes:
https://github.com/Navid200/xDrip/wiki/Native-Algorithm

Considering you will be giving up factory calibration, it's not recommended to use non-native mode if you have  G6.
But, you will have to put up with some sensitivities the transmitter has (explained before) when you use native mode.

**Reply by hgreyh** [reply]
Thank you.  how can i distinguish an older transmitter from another?

**Reply by hgreyh** [reply]
Thank you.  Why was the calibration accepted when the sensor was started and OB1 was disabled?  the BG value has been changed from 2.6 to 3.4 mmol.

**Reply by Navid200** [reply]
I suspect you are running in non-native mode.
Do you mind showing a screenshot of the following page?

Settings -> G5/G6 Debug Settings

If the second character of the serial number is either 0 or 1, it is an old transmitter.
If the firmware version is 1.6.5.25 or earlier, you can hard reset the transmitter.

**Reply by hgreyh** [reply]
this is what i did: 1.replaced the sensor; 2.started sensor; 3. got the error "clock not syncronized ..."; 4.disabled OB1; 5. started the sensor; 6.entered two calibration values ​​requested by xdrip.
![20220106_230931](https://user-images.githubusercontent.com/74299959/148448320-caa4c992-c0e1-4e4c-9367-f3bc5ff9f1a9.jpg)
![20220106_231111](https://user-images.githubusercontent.com/74299959/148448337-d09e7bd1-29d9-45cb-9a6e-115758b4747e.jpg)

**Reply by Navid200** [reply]
I see.
By disabling the OB1 collector, you have effectively taken away the ability to use the native mode.
This is a very old way to use xDrip.  I have been using xDrip for more than 3 years now and I have never done this.

If I were you, I would enable it when the time comes to start the next sensor.
After you stop sensor and remove the old sensor, as well as the transmitter, from your body, set a timer to 10 minutes.
By the time you clean your skin and insert the new sensor and clean the transmitter, the timer should run off.
Only after the timer runs its course, should you snap the transmitter into the new sensor.
If you now start the sensor, there should be no issues with starting the sensor.

**Comment by tolot27** [comment]
Please use the Dexcom Receiver or Dexcom App to try to start the sensor.
